src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -271,6 +307,9 @@ function BulkEmailForm(props) {
           handleCheckboxes={onRecipientChange}
           additionalCohorts={cohorts}
           isValid={emailFormValidation.recipients}
+          emailLearnersList={emailLearnersList}
+          handleLearnersEmailSelected={handleEmailLearnersSelected}
+          handleLearnersDeleteEmail={handleDeleteEmailLearnerSelected}
```

Comment:
I'd suggest (as previously) to change `EmailLearners` with `EmailIndividualStudents`